package/MDAnalysis/core/groups.py

```diff
@@ -1581,7 +1655,7 @@ def wrap(self, compound="atoms", center="com", box=None, inplace=True):
         # atoms:
         atoms = self.atoms
         if not self.isunique:
-            _atoms = atoms.unique
+            _atoms = atoms.sorted_unique
```

Comment:
State explicitly in the docs that atoms will be sorted? (Makes sense to me.)

Btw, can you please fix the docs a few lines above
```reST
:meth:`MDanalysis.lib.distances.apply_PBC`
```
to
```reST
:meth:`MDAnalysis.lib.distances.apply_PBC`
```

Reply:
I don't think it matters, as below the positions get re-ordered back to the input.

```
if not self.isunique:
	positions = positions[restore_mask]
```